### PR TITLE
Add test for repo sets in new host details ui

### PR DESCRIPTION
**Pull request by peterdragun**
- depends on SatelliteQE/airgun/pull/770

### CODE REVIEW

**Comment by peterdragun** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k "test_positive_update_delete_package or test_positive_apply_erratum or test_positive_crud_module_streams"
airgun: 770

**Comment by peterdragun** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k "(test_positive_update_delete_package or test_positive_apply_erratum or test_positive_crud_module_streams)"
airgun: 770

**Comment by peterdragun** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k '(test_positive_update_delete_package or test_positive_apply_erratum or test_positive_crud_module_streams)'
airgun: 770

**Comment by peterdragun** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k '(test_positive_update_delete_package or test_positive_apply_erratum or test_positive_crud_module_streams)'
airgun: 770

**Comment by peterdragun** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -v -k '(test_positive_update_delete_package or test_positive_apply_erratum or test_positive_crud_module_streams)'
airgun: 770

**Comment by peterdragun** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -v -k '(test_positive_update_delete_package or test_positive_apply_erratum or test_positive_crud_module_streams)'
airgun: 770

**Comment by peterdragun** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -v -k '(test_positive_update_delete_package or test_positive_apply_erratum or test_positive_crud_module_streams)'
airgun: 770

**Comment by peterdragun** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -v -k '(test_positive_update_delete_package or test_positive_apply_erratum or test_positive_crud_module_streams)'
airgun: 770

**Comment by peterdragun** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -v -k test_positive_apply_erratum
airgun: 770

**Comment by peterdragun** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -v -k '(test_positive_update_delete_package or test_positive_apply_erratum or test_positive_crud_module_streams)'
airgun: 770

**Comment by peterdragun** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -v -k '(test_positive_update_delete_package or test_positive_apply_erratum or test_positive_crud_module_streams)'
airgun: 770

**Comment by peterdragun** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -v -k '(test_positive_update_delete_package or test_positive_apply_erratum or test_positive_crud_module_streams)'
airgun: 770

**Comment by peterdragun** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -v -k '(test_positive_update_delete_package or test_positive_apply_erratum or test_positive_crud_module_streams)'
airgun: 770

**Comment by peterdragun** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -v -k '(test_positive_update_delete_package or test_positive_apply_erratum or test_positive_crud_module_streams)'
airgun: 770

**Review by Gauravtalreja1 — APPROVED** [review]

File: tests/foreman/ui/test_host.py [hunk 1]
```diff
@@ -2219,32 +2220,54 @@ def test_positive_update_delete_package(
...
     with session:
         session.location.select(loc_name=DEFAULT_LOC)
+        if not is_open('BZ:2132680'):
+            product_name = module_repos_collection_with_setup.custom_product.name
+            repos = session.host_new.get_repo_sets(client.hostname, product_name)
+            assert 'Enabled' == repos[0].status
+            session.host_new.override_repo_sets(
+                client.hostname, product_name, "Override to disabled"
+            )
+            assert 'Disnabled' == repos[0].status
```
[the remark this answers is not in the document]
Comment (ColeHiggins2) [reply tests/foreman/ui/test_host.py:2257]: Looks like we are skipping this block due to the BZ being open. Probably will fail once BZ is closed

[the remark this answers is not in the document]
Comment (peterdragun) [reply tests/foreman/ui/test_host.py:2257]: Oh good catch. Yes I did. I will fix it ASAP

[the remark this answers is not in the document]
Comment (peterdragun) [reply tests/foreman/ui/test_host.py:2257]: Fixed in #10368

**Pull request merged**